keyboards/hfdkb/8s200/config.h

```diff
@@ -0,0 +1,43 @@
+/* Copyright (C) 2022 Westberry Technology (ChangZhou) Corp., Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+/* I2C Config for LED Driver */
+#define IS31FL3733_DRIVER_COUNT 2
+#define IS31FL3733_I2C_ADDRESS_1 0b1110100
+#define IS31FL3733_I2C_ADDRESS_2 0b1110111
```

Comment:
These do not look like valid address per the datasheet, nor are they using the correct defines.

This is only a guess at A6 being incorrectly set to 1 instead of 0.....
```suggestion
#define IS31FL3733_I2C_ADDRESS_1 IS31FL3733_I2C_ADDRESS_SCL_GND
#define IS31FL3733_I2C_ADDRESS_2 IS31FL3733_I2C_ADDRESS_SCL_VCC
#define IS31FL3733_SDB_PIN A15


```

Reply:
Sorry, I mixed up the LED control IC. According to the actual PCB, it’s actually using the SNLED27351, so its actual address starts with 0x7.